### ISSUE TITLE
Creating nested droppable area dynamically

### ISSUE THREAD

**Issue by ozanbatuhanceylan** [akserg/ng2-dnd#186]
Hi. First of all thanks for this great component. I have a question about nested droppable area. I'm sorry if same question was asked before.

I have some draggable objects and have a drop zone for these. I want to make these draggable objects droppable when object is dropped. So i can create nested droppable fields dynamically. 

Is it possible? If it's possible how can i do this?

**Comment by VarshilDoshi** [comment]
yes, i want to create dropzones dynamically,like when i drop any objects in dropzones, objects automatically make droppable objects..
